### DISCUSSION TITLE
More frequent releases

### DISCUSSION THREAD

**Discussion by mKaloer** [ory/hydra#3854] [Ideas]
Hi,

We would greatly appreciate if you can publish new releases more frequently. This is a fairly critical piece of software, so we (and likely others as well) have policies in place to fix certain vulnerabilities in our code. Often, these vulnerabilities are found in your dependencies. 
Since you actually bump dependencies to the new versions frequently in the source code, but don't release often, It would be really great if you could increase the frequency of patch releases.

Thanks!

**Comment by aeneasr** [comment]
Hello @mKaloer - you will find more information about this use case here: https://github.com/ory/hydra/security/policy

**Reply by mKaloer** [reply]
Thanks, I see – so there is no way around this if we want a self-hosted solution, unless we go for an enterprise license?

**Reply by aeneasr** [reply]
We only provide an SLA for addressing CVEs with an enterprise license agreement (that can be self-hosted). Without a commercial agreement we do not offer a security SLA.

**Reply by mKaloer** [reply]
Okay, we will make a new build ourselves then. Thanks for your answers.
